tests/test_fixtures/rot13_base64_hex_with_newline

```diff
@@ -0,0 +1 @@
+52 33 56 32 5a 69 42 32 5a 69 42 75 49 47 64 79 5a 6d 63 68
```

Comment:
You'll notice GitHub does not display the new line but complains about the file above not having one, this is another quirk of Unix 😂